examples/CFD/airfoil3d.py

```diff
@@ -32,7 +32,6 @@
 import matplotlib.pylab as plt
 from src.models import BGKSim, KBCSim
 from src.boundary_conditions import *
-from src.lattice import *
```

Comment:
I think lattice definition should be explicit inside all examples. That's just good practice and does not make the implication that BGK can only work with D3Q19